src/services/types.ts

```diff
@@ -323,6 +323,7 @@ namespace ts {
         getDocCommentTemplateAtPosition(fileName: string, position: number): TextInsertion | undefined;
 
         isValidBraceCompletionAtPosition(fileName: string, position: number, openingBrace: number): boolean;
+        getAutoCloseTagAtPosition(fileName: string, position: number): string | undefined;

```

Comment:
i would rather this had `JSX` in the name.. like `getJSXClosingTagAtPosition` also we need to add a comment on it when it should be called.